### DISCUSSION TITLE
Multiple server?

### DISCUSSION THREAD

**Discussion by jlcmux** [evertramos/nginx-proxy-automation#384] [Q&A]
I have prod and develop servers, current prod have nginx-proxy-automation working well and i would like to publish develop versions with other domains but i have only one public IP. can this app generate certs and redirect traffic to other servers?, can i create manual entries on reverse proxy ?

**Comment by jlcmux** [comment]
For example /etc/nginx/conf.d/default.conf add custom entry but maybe it will be overwrite  and not sure how to generate the certificate.

**Reply by lschweiss** [reply]
Did you ever figure this out?  I came here trying to do precisely this.
